modules/appeals_api/spec/requests/v2/contestable_issues_controller_spec.rb

```diff
@@ -4,42 +4,24 @@
 require_relative AppealsApi::Engine.root.join('spec', 'support', 'shared_examples_contestable_issues.rb')
 
 describe AppealsApi::V2::DecisionReviews::ContestableIssuesController, type: :request do
-  include_examples 'contestable issues index requests',
-                   decision_review_type: 'higher_level_reviews',
-                   benefit_type: 'compensation',
-                   version: 'v2'
+  include_examples 'Contestable Issues API v0 and Decision Reviews v1 & v2 shared request examples',
+                   base_path: '/services/appeals/v2/decision_reviews/contestable_issues'
 
-  describe 'using versioned namespace route with oauth' do
-    include_examples 'contestable issues index requests',
-                     decision_review_type: 'higher_level_reviews',
-                     benefit_type: 'compensation',
-                     use_versioned_namespace_route: true,
```

Comment:
Previously, we were initiating tests of the v0 API from here, in the decision reviews v2 tests - I moved this to the dedicated spec file for the v0 controller instead